external/netstandard/netstandard.depproj

```diff
@@ -35,6 +35,12 @@
     <PackageReference Condition="'$(NETStandardVersion)' &gt;= 1.2" Include="System.Data.SqlClient">
       <Version>4.3.0</Version>
     </PackageReference>
+    <PackageReference Condition="'$(NETStandardVersion)' &gt;= 1.3" Include="System.Security.Cryptography.Cng">
```

Comment:
I see, you aren't building the old reference assemblies and we need those to generate the platform not supported assembly.  In that case please just include one item with `&gt;= 1.3` that should be sufficient.